systemd/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set(CEPH_SYSTEMD_ENV_DIR "/etc/sysconfig"
 set(SYSTEMD_ENV_FILE "${CEPH_SYSTEMD_ENV_DIR}/ceph")
 foreach(service
     ceph-crash
+    ceph-exporter@
```

Comment:
@peng225 we are close. just one (last) question, do we need to run multiple instances of ceph-exporter on a single host? as it enumerate all asock files under the configured path which is presumably shared by these files. so i guess a single instance would suffice. am i right?

Reply:
@tchaikov Oops, I think you are right.
I removed the `@` sign.